tensorflow/compiler/aot/tfcompile.bzl

```diff
@@ -23,7 +23,7 @@ def tf_library(name, graph, config,
                cpp_class=None, gen_test=True, gen_benchmark=True,
                visibility=None, testonly=None,
                tfcompile_flags=None,
-               tfcompile_tool="//tensorflow/compiler/aot:tfcompile",
+               tfcompile_tool=str(Label("//tensorflow/compiler/aot:tfcompile")),
```

Comment:
Can you use the clean_dep() wrapper defined here:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tensorflow.bzl#L49

Same everywhere.